tests/file_uploads/tests.py

```diff
@@ -480,8 +480,8 @@ def test_file_error_blocking(self):
         """
         class POSTAccessingHandler(client.ClientHandler):
             """A handler that'll access POST during an exception."""
-            def handle_uncaught_exception(self, request, resolver, exc_info):
-                ret = super(POSTAccessingHandler, self).handle_uncaught_exception(request, resolver, exc_info)
+            def handle_uncaught_exception(self, request, exc_info):
```

Comment:
Florian, while reviewing/rebasing this and trying to extract some of the handler restructuring into a standalone commit to make reviewing the rest of this a bit easier, I noticed that the `Handler.handle_uncaught_exception()` method is removed so this method here isn't called anymore. I'm not sure if this test needs to be updated or if it can be removed but I wanted to make sure you were aware of this and if so, could you explain the rationale so I can document it in the release notes?